project/Snapper/SnapperSettings.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using Newtonsoft.Json;
+
+namespace Snapper
+{
+    public static class SnapperSettings
+    {
+        public static Func<JsonSerializerSettings> SnapshotDeserializationSettings { get; set; } = () => null;

```

Comment:
This extra setting allows to modify any serialization settings without touching default `JsonConvert.DefaultSettings` - can be handy in case of using the setting already.